hack/ci/e2e-k8s.sh

```diff
@@ -115,7 +115,9 @@ create_cluster() {
       apiServer_extra_args="${apiServer_extra_args}
       \"logging-format\": \"${CLUSTER_LOG_FORMAT}\""
   fi
-  kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  kubelet_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\"
+      \"container-log-max-files\": \"10\"
+      \"container-log-max-size\": \"5Mi\""
```

Comment:
still not enough 🤔 

https://gcsweb.k8s.io/gcs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kind/3774/pull-kind-e2e-kubernetes-1-30/1852821515305750528/artifacts/kind-control-plane/pods/kube-system_kube-apiserver-kind-control-plane_9c7fae38e19217590379c7ebcfd2e100/kube-apiserver/

The max files seems to be respected, but the max size does not, can you set this to 100Mi?
```suggestion
      \"container-log-max-size\": \"100Mi\""
```